scripts/release/atomic_pipeline.py

```diff
@@ -419,32 +416,25 @@ def queue_exception_handling(tasks_queue):
         )
 
 
-def _build_agent_operator(
+def _build_agent(
```

Comment:
this is just a - for now - fix for the pipeline, next version is a proper multi-arch solution for atomic